packages/tizen_bundle/.metadata

```diff
@@ -0,0 +1,27 @@
+# This file tracks properties of this Flutter project.
```

Comment:
Please remove this file as it's not expected to be included in this repo.

https://github.com/flutter-tizen/flutter-tizen/wiki/New-plugin-submission-guideline

Reply:
Done.